Automattic/jetpack
# Payment Buttons: Add vertical margin dimension controls

Partly addresses #25927 - blockGap (block spacing) is not implemented in this PR, just vertical margins.

#### Changes proposed in this Pull Request:

Add vertical margin controls for the payment-buttons block (used to group recurring-payment buttons) in the dimensions panel. 

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Get a site on a paid JP/wpcom plan
* Insert a Payment Buttons block
* Go through the stripe set-up
* When returned to the editor, publish the page and view it
* Apply this patch / use this branch with jetpack beta on a JN/atomic site
* Refresh the page opened earlier - it should be unchanged
* Edit the block by changing the text etc
* Save the page
* View the page - it should look identical
* Go back to Gutenberg
* Select the payments-buttons block
* Open the sidebar and press plus on dimensions add margin settings
* Play with margins and notice the buttons change appropriately - the vertical margins only changing
* Save the page and review it - notice the margins settings have taken effect.
* Add a premium-content block, notice the buttons are unaffected.


https://user-images.githubusercontent.com/93301/188712559-b87214fc-3adc-4828-a100-f14ef282bf52.mp4
